### ISSUE TITLE
Unlabeled image(s) exposed in Welcome to Thorium 3.0 dialogue

### ISSUE THREAD

**Issue by togregg** [edrlab/thorium-reader#2614]
### Impact: Major
[WCAG 2.1 SC 1.1.1 – Non-Text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content)

### Current Behavior:
When you are in the Welcome to Thorium 3.0 dialogue and you are navigating through the Home, Your Publications, Catalogs, Reading View and Annotations tabs, the image for each area is announced as unlabeled by Voice Over. Just the image on the Home tab is exposed to JAWS and NVDA and is announced as unlabeled.

### Example:
Using VoiceOver and focus is in the Welcome to Thorium 3.0 dialogue, from the Home tab press the Tab key to go into the content. Then use VO + Right arrow to navigate the text and when you get to the image, Voice over announces “To get missing image descriptions, open the context menu., Unlabeled image.”

### Expected Behavior:
When you are in the Welcome to Thorium 3.0 dialogue and you are navigating through the Home, Your Publications, Catalogs, Reading View and Annotations tabs, the image for each area is ignored by Voice Over. The image for the Home tab is also ignored by JAWS and NVDA, just like all the other images in the dialogue.

#### Windows
- Windows 11 Pro Version 22H2 OS build 22621.3447
- JAWS 2024.2409.2
- NVDA 2024.1 (2024.1.0.31547)
- Thorium v3.0.0

#### macOS
- Sonoma 14.7 (23H124)
- Voice Over for Sonoma
- Thorium 3.0.0

### Steps to recreate:

1. Launch screen reader.
2. Open Thorium.
3.  From the Welcome to Thorium 3.0 dialogue use the VO + right arrow (VoiceOver) or down arrow (JAWS/NVDA) to navigate to the image under the Home tab.
4. Screen reader announces image as unlabeled.
5. Just using VoiceOver, press Shift Tab twice to get to Home tab.
6. Press Down arrow to go to the Your Publications Tab and then press Tab twice to land on the close button.
7. Use VO + right arrow to get to the image and VoiceOver announces the file name and then unlabeled image.
8. Repeat this process for the other tabs to hear VoiceOver announce the images as unlabeled.

### Video(s)/Screenshots:
(Screenshot of Thorium welcome dialogue with VoiceOver announcing image under the Home tab as unlabeled)
![image](https://github.com/user-attachments/assets/4b1c8ca6-646a-445c-bbf4-eacddd1eb75c)

### Recommendations:

- If the intent of the images are to be decorative, then the images in the Welcome to Thorium 3.0 dialogue should be coded to be ignored by all screen readers. 
- If they are meant to be visible, then all of the images need alt text.

**Comment by danielweck** [comment]
Hello @togregg I apologise for my lack of feedback, just a quick message to let you know that we really appreciate your constructive analysis and suggested improvements. Thank you!
